src/TabContent.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes, Component } from 'react';
+import classnames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  activeTab: PropTypes.any,
+  className: PropTypes.string
+};
+
+const childContextTypes = {
+  activeTabId: PropTypes.any
+};
+
+export default class TabContent extends Component {
+  constructor(props) {
+    super(props);
+    this.childRefs = {};
+    this.state = {
+      activeTab: this.props.activeTab
+    };
+  }
+  getChildContext() {
+    return {
+      activeTabId: this.state.activeTab
+    };
+  }
+  componentWillReceiveProps(nextProps) {
+    this.setState({
```

Comment:
Not sure why github is not collapsing or updating this comment but this is fixed